core-metadata/pom.xml

```diff
@@ -87,6 +99,10 @@
             <groupId>com.tdunning</groupId>
             <artifactId>t-digest</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.haifengl</groupId>
+            <artifactId>smile-core</artifactId>
```

Comment:
Could you please add the copyright of smile-core to Kylin's LICENSE file in the root folder?